source/Game.cpp

```diff
@@ -135,6 +148,10 @@ bool Game::frameRenderingQueued(Ogre::FrameEvent const &fe) {
   // Need to capture / update each device
   Keyboard::getKeyboard()->capture();
   Mouse::getMouse()->capture();
+  for (auto it = Joystick::getJoysticks().begin(); it != Joystick::getJoysticks().end(); it++)
```

Comment:
Consider using : http://en.cppreference.com/w/cpp/language/range-for
it makes code more readable.
Ex:
```C++
for (A &a : as)
  // do stuff
```